diff -uprN FTL-5.1/src/CMakeLists.txt FTL-5.1.cust/src/CMakeLists.txt
--- FTL-5.1/src/CMakeLists.txt	2020-07-15 17:26:33.000000000 -0400
+++ FTL-5.1.cust/src/CMakeLists.txt	2020-07-19 11:53:17.954159522 -0400
@@ -172,10 +172,10 @@ set(CMAKE_THREAD_PREFER_PTHREAD TRUE)
 set(THREADS_PREFER_PTHREAD_FLAG TRUE)
 find_package(Threads REQUIRED)
 
-find_library(HOGWEED libhogweed${CMAKE_STATIC_LIBRARY_SUFFIX})
-find_library(GMP libgmp${CMAKE_STATIC_LIBRARY_SUFFIX})
-find_library(NETTLE libnettle${CMAKE_STATIC_LIBRARY_SUFFIX})
-find_library(IDN libidn${CMAKE_STATIC_LIBRARY_SUFFIX})
+find_library(HOGWEED libhogweed${CMAKE_SHARED_LIBRARY_SUFFIX})
+find_library(GMP libgmp${CMAKE_SHARED_LIBRARY_SUFFIX})
+find_library(NETTLE libnettle${CMAKE_SHARED_LIBRARY_SUFFIX})
+find_library(IDN libidn${CMAKE_SHARED_LIBRARY_SUFFIX})
 
 # for FTL we need the pthread library
 # for dnsmasq we need the nettle crypto library and the gmp maths library
diff -uprN FTL-5.1/src/config.c FTL-5.1.cust/src/config.c
--- FTL-5.1/src/config.c	2020-07-15 17:26:33.000000000 -0400
+++ FTL-5.1.cust/src/config.c	2020-07-19 11:53:17.960826066 -0400
@@ -57,7 +57,7 @@ void getLogFilePath(void)
 	}
 
 	// Read LOGFILE value if available
-	// defaults to: "/var/log/pihole-FTL.log"
+	// defaults to: "/run/log/pihole-ftl/pihole-FTL.log"
 	buffer = parse_FTLconf(fp, "LOGFILE");
 
 	errno = 0;
@@ -65,7 +65,7 @@ void getLogFilePath(void)
 	if(buffer == NULL || sscanf(buffer, "%127ms", &FTLfiles.log) != 1)
 	{
 		// Use standard path if no custom path was obtained from the config file
-		FTLfiles.log = strdup("/var/log/pihole-FTL.log");
+		FTLfiles.log = strdup("/run/log/pihole-ftl/pihole-FTL.log");
 	}
 
 	// Test if memory allocation was successful
@@ -298,10 +298,10 @@ void read_FTLconf(void)
 		logg("   DBIMPORT: Not importing history from database");
 
 	// PIDFILE
-	getpath(fp, "PIDFILE", "/run/pihole-FTL.pid", &FTLfiles.pid);
+	getpath(fp, "PIDFILE", "/run/pihole-ftl-FTL.pid", &FTLfiles.pid);
 
 	// SOCKETFILE
-	getpath(fp, "SOCKETFILE", "/run/pihole/FTL.sock", &FTLfiles.socketfile);
+	getpath(fp, "SOCKETFILE", "/run/pihole-ftl/FTL.sock", &FTLfiles.socketfile);
 
 	// SETUPVARSFILE
 	getpath(fp, "SETUPVARSFILE", "/etc/pihole/setupVars.conf", &FTLfiles.setupVars);
diff -uprN FTL-5.1/src/version.h.in FTL-5.1.cust/src/version.h.in
--- FTL-5.1/src/version.h.in	2020-07-15 17:26:33.000000000 -0400
+++ FTL-5.1.cust/src/version.h.in	2020-07-19 11:53:17.950826250 -0400
@@ -1,11 +1,11 @@
 #ifndef VERSION_H
 #define VERSION_H
 
-#define GIT_VERSION "@GIT_VERSION@"
-#define GIT_DATE "@GIT_DATE@"
-#define GIT_BRANCH "@GIT_BRANCH@"
-#define GIT_TAG "@GIT_TAG@"
-#define GIT_HASH "@GIT_HASH@"
+#define GIT_VERSION "5.1"
+#define GIT_DATE "2020-07-19"
+#define GIT_BRANCH "master"
+#define GIT_TAG "5.1"
+#define GIT_HASH "builtfromreleasetarball"
 #define FTL_ARCH "@FTL_ARCH@"
 #define FTL_CC "@FTL_CC@"
 
diff -uprN FTL-5.1/tools/socket_client.c FTL-5.1.cust/tools/socket_client.c
--- FTL-5.1/tools/socket_client.c	2020-07-15 17:26:33.000000000 -0400
+++ FTL-5.1.cust/tools/socket_client.c	2020-07-19 11:53:17.964159338 -0400
@@ -40,7 +40,7 @@ int main (int argc, char **argv) {
 	address.sun_family = AF_LOCAL;
 
 	char *command = ">stats";
-	strcpy(address.sun_path,"/run/pihole/FTL.sock");
+	strcpy(address.sun_path,"/run/pihole-ftl/FTL.sock");
 
 	int i;
 	for(i = 1; i < argc; i++) {
