diff -uprN FTL-5.8/src/CMakeLists.txt FTL-5.8.cust/src/CMakeLists.txt
--- FTL-5.8/src/CMakeLists.txt	2021-04-14 19:52:20.000000000 +0200
+++ FTL-5.8.cust/src/CMakeLists.txt	2021-04-15 21:06:33.685015890 +0200
@@ -81,7 +81,7 @@ if(CMAKE_C_COMPILER_VERSION VERSION_EQUA
 else()
     set(EXTRAWARN_GCC8 "")
 endif()
-set(EXTRAWARN "-Werror -Waddress -Wlogical-op -Wmissing-field-initializers -Woverlength-strings -Wformat -Wformat-nonliteral -Wuninitialized -Wswitch-enum -Wshadow -Wfloat-equal -Wbad-function-cast -Wwrite-strings -Wparentheses -Wlogical-op -Wstrict-prototypes -Wmissing-prototypes -Wredundant-decls -Winline ${EXTRAWARN_GCC8}")
+set(EXTRAWARN "-Waddress -Wlogical-op -Wmissing-field-initializers -Woverlength-strings -Wformat -Wformat-nonliteral -Wuninitialized -Wswitch-enum -Wshadow -Wfloat-equal -Wbad-function-cast -Wwrite-strings -Wparentheses -Wlogical-op -Wstrict-prototypes -Wmissing-prototypes -Wredundant-decls -Winline ${EXTRAWARN_GCC8}")
 separate_arguments(EXTRAWARN)
 
 # Do we want to compile a statically linked musl executable?
@@ -191,10 +191,10 @@ set(THREADS_PREFER_PTHREAD_FLAG TRUE)
 find_package(Threads REQUIRED)
 
 # for DNSSEC we need the nettle (+ hogweed) crypto and the gmp math libraries
-find_library(LIBHOGWEED libhogweed${CMAKE_STATIC_LIBRARY_SUFFIX})
-find_library(LIBGMP libgmp${CMAKE_STATIC_LIBRARY_SUFFIX})
-find_library(LIBNETTLE libnettle${CMAKE_STATIC_LIBRARY_SUFFIX})
-find_library(LIBIDN libidn${CMAKE_STATIC_LIBRARY_SUFFIX})
+find_library(LIBHOGWEED libhogweed${CMAKE_SHARED_LIBRARY_SUFFIX})
+find_library(LIBGMP libgmp${CMAKE_SHARED_LIBRARY_SUFFIX})
+find_library(LIBNETTLE libnettle${CMAKE_SHARED_LIBRARY_SUFFIX})
+find_library(LIBIDN libidn${CMAKE_SHARED_LIBRARY_SUFFIX})
 
 target_link_libraries(pihole-FTL rt Threads::Threads ${LIBHOGWEED} ${LIBGMP} ${LIBNETTLE} ${LIBIDN})
 
@@ -203,9 +203,9 @@ if(LUA_DL STREQUAL "true")
     target_link_libraries(pihole-FTL ${LIBDL})
 endif()
 
-find_library(LIBREADLINE libreadline${CMAKE_STATIC_LIBRARY_SUFFIX})
-find_library(LIBHISTORY libhistory${CMAKE_STATIC_LIBRARY_SUFFIX})
-find_library(LIBTERMCAP libtermcap${CMAKE_STATIC_LIBRARY_SUFFIX})
+find_library(LIBREADLINE libreadline${CMAKE_SHARED_LIBRARY_SUFFIX})
+find_library(LIBHISTORY libhistory${CMAKE_SHARED_LIBRARY_SUFFIX})
+find_library(LIBTERMCAP libtermcap${CMAKE_SHARED_LIBRARY_SUFFIX})
 if(LIBREADLINE AND LIBHISTORY AND LIBTERMCAP)
     message(STATUS "Building FTL with readline support: YES")
     target_compile_definitions(FTL PRIVATE LUA_USE_READLINE)
diff -uprN FTL-5.8/src/config.c FTL-5.8.cust/src/config.c
--- FTL-5.8/src/config.c	2021-04-14 19:52:20.000000000 +0200
+++ FTL-5.8.cust/src/config.c	2021-04-15 21:06:33.693015940 +0200
@@ -64,7 +64,7 @@ void getLogFilePath(void)
 	}
 
 	// Read LOGFILE value if available
-	// defaults to: "/var/log/pihole-FTL.log"
+	// defaults to: "/run/log/pihole-ftl/pihole-FTL.log"
 	buffer = parse_FTLconf(fp, "LOGFILE");
 
 	errno = 0;
@@ -72,7 +72,7 @@ void getLogFilePath(void)
 	if(buffer == NULL || sscanf(buffer, "%127ms", &FTLfiles.log) != 1)
 	{
 		// Use standard path if no custom path was obtained from the config file
-		FTLfiles.log = strdup("/var/log/pihole-FTL.log");
+		FTLfiles.log = strdup("/run/log/pihole-ftl/pihole-FTL.log");
 	}
 
 	// Test if memory allocation was successful
@@ -318,14 +318,14 @@ void read_FTLconf(void)
 		logg("   DBIMPORT: Not importing history from database");
 
 	// PIDFILE
-	getpath(fp, "PIDFILE", "/run/pihole-FTL.pid", &FTLfiles.pid);
+	getpath(fp, "PIDFILE", "/run/pihole-ftl/pihole-FTL.pid", &FTLfiles.pid);
 
 	// PORTFILE
-	getpath(fp, "PORTFILE", "/run/pihole-FTL.port", &FTLfiles.port);
+	getpath(fp, "PORTFILE", "/run/pihole-ftl/pihole-FTL.port", &FTLfiles.port);
 	saveport(config.port);
 
 	// SOCKETFILE
-	getpath(fp, "SOCKETFILE", "/run/pihole/FTL.sock", &FTLfiles.socketfile);
+	getpath(fp, "SOCKETFILE", "/run/pihole-ftl/FTL.sock", &FTLfiles.socketfile);
 
 	// SETUPVARSFILE
 	getpath(fp, "SETUPVARSFILE", "/etc/pihole/setupVars.conf", &FTLfiles.setupVars);
diff -uprN FTL-5.8/src/version.h.in FTL-5.8.cust/src/version.h.in
--- FTL-5.8/src/version.h.in	2021-04-14 19:52:20.000000000 +0200
+++ FTL-5.8.cust/src/version.h.in	2021-04-15 21:06:33.680015858 +0200
@@ -1,11 +1,11 @@
 #ifndef VERSION_H
 #define VERSION_H
 
-#define GIT_VERSION "@GIT_VERSION@"
-#define GIT_DATE "@GIT_DATE@"
-#define GIT_BRANCH "@GIT_BRANCH@"
-#define GIT_TAG "@GIT_TAG@"
-#define GIT_HASH "@GIT_HASH@"
+#define GIT_VERSION "v5.8"
+#define GIT_DATE "2021-04-15"
+#define GIT_BRANCH "master"
+#define GIT_TAG "v5.8"
+#define GIT_HASH "builtfromreleasetarball"
 #define FTL_ARCH "@FTL_ARCH@"
 #define FTL_CC "@FTL_CC@"
 
diff -uprN FTL-5.8/tools/socket_client.c FTL-5.8.cust/tools/socket_client.c
--- FTL-5.8/tools/socket_client.c	2021-04-14 19:52:20.000000000 +0200
+++ FTL-5.8.cust/tools/socket_client.c	2021-04-15 21:06:33.694015946 +0200
@@ -40,7 +40,7 @@ int main (int argc, char **argv) {
 	address.sun_family = AF_LOCAL;
 
 	char *command = ">stats";
-	strcpy(address.sun_path,"/run/pihole/FTL.sock");
+	strcpy(address.sun_path,"/run/pihole-ftl/FTL.sock");
 
 	int i;
 	for(i = 1; i < argc; i++) {
